### ISSUE TITLE
Flash of "no theme"

### ISSUE THREAD

**Issue by tomgidden** [react-theming/storybook-addon#4]
Hi,

Whenever a story renders, I'm seeing a flash of "no theme" for a split-second before it works.

This is just a small annoyance in normal use, but Chromatic seems to screenshot the "no theme", so it breaks that facility completely.

I've tracked it down to `src/preview/index.js`, occurring when `isFirstDataReceived` is falsey, which comes from `@storybook/addon-devkit`.  I don't know where to go from there.

This is with:
* @storybook/react v5.3.19
* @react-theming/storybook-addon v1.0.2
* @storybook/addon-devkit v1.3.7
* @material-ui/core v4.11.0

and calling
```js
const themeProvider = ({theme, children}) => (
    <MuiThemeProvider theme={createMuiTheme(theme)}>
        {children}
    </MuiThemeProvider>
);
addDecorator(withThemes(themeProvider, [themeData]))
```
in `.storybook/preview.js`

I suppose this could be a similar problem to that in #3, considering the indication of "no theme".

**Comment by sashapawcloud** [comment]
Moreover, snapshots look like:
```
// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/Inputs/Button Default 1`] = `
<p>
  no theme
</p>
`;

exports[`Storyshots Components/Inputs/Button Disabled 1`] = `
<p>
  no theme
</p>
`;
```

**Comment by UsulPro** [comment]
Fixed in #5 
update to v1.0.3
